.pipelines/android-daily.yml

```diff
@@ -57,11 +40,19 @@ steps:
     tasks: 'adaptivecards:build'
 
 - task: Gradle@2
-  displayName: 'gradlew publish'
+  displayName: 'Gradle Unsigned Publish Local'
```

Comment:
is this file expected to be in the JS pipeline?

Reply:
It is just a minor refactor, to match the recent change to android-release.yml. I could probably create a separate PR for it, if necessary